fdbserver/include/fdbserver/ProxyCommitData.actor.h

```diff
@@ -277,6 +277,8 @@ struct ProxyCommitData {
 	std::shared_ptr<AccumulativeChecksumBuilder> acsBuilder = nullptr;
 	LogEpoch epoch;
 
+	Version lastShardMove;
+
```

Comment:
Where is this getting populated please? Is that done in a different commit?

Reply:
the constructor initializes it to invalidVersion.